tracing-subscriber/README.md

```diff
@@ -32,6 +32,67 @@ Utilities for implementing and composing [`tracing`][tracing] subscribers.
 [discord-url]: https://discordapp.com/invite/XdPzyTZ
 [maint-badge]: https://img.shields.io/badge/maintenance-experimental-blue.svg
 
+## Usage
+
+First, add this to your `Cargo.toml` file:
+
+```toml
+[dependencies]
+tracing-subscriber = "0.1"
+```
+
+Add the following to your executable to initialize the default subscriber:
+```rust
+use tracing_subscriber;
+
+tracing_subscriber::fmt::init();
+```
+
+**Note**: This should **not** be called by libraries. Libraries should use [`tracing`](../tracing) to publish `tracing` `Event`s.
+
+## Configuration
+
+You can configure a subscriber instead of using the defaults with the following functions:
+
+### Subscriber
+
+The `FmtSubscriber` is currently the recommended `Subscriber`. You can create one by calling:
```

Comment:
I don't really care for this language. Instead of "`FmtSubscriber` is currently the recommended `Subscriber`", can we explain what `FmtSubscriber` _does_? It would be good to say that it implements a subscriber that records traces as line-oriented logs. Other use-cases will call for other subscriber implementations.